dev/ProgressBar/ProgressBar.xaml

```diff
@@ -107,19 +113,45 @@
                             contract7NotPresent:CornerRadius="{ThemeResource ControlCornerRadius}">
 
                             <Border Clip="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.ClipRect}">
-                                <Rectangle x:Name="ProgressBarIndicator"
-                                    Margin="{TemplateBinding Padding}"
-                                    Fill="{TemplateBinding Foreground}"         
-                                    HorizontalAlignment="Left"
-                                    contract7Present:RadiusX="{Binding CornerRadius, RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource TopLeftCornerRadiusDoubleValueConverter}}"
-                                    contract7Present:RadiusY="{Binding CornerRadius, RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource BottomRightCornerRadiusDoubleValueConverter}}"
-                                    contract7NotPresent:RadiusX="{Binding Source={ThemeResource ControlCornerRadius}, Converter={StaticResource TopLeftCornerRadiusDoubleValueConverter}}"
-                                    contract7NotPresent:RadiusY="{Binding Source={ThemeResource ControlCornerRadius}, Converter={StaticResource BottomRightCornerRadiusDoubleValueConverter}}"
-    >
-                                    <Rectangle.RenderTransform>
-                                        <CompositeTransform/>
-                                    </Rectangle.RenderTransform>
-                                </Rectangle>
+                                <StackPanel Orientation="Horizontal">
```

Comment:
>StackPanel [](start = 33, length = 10)

I think you should use a grid instead, that way you wont need to add render transforms to get the children to overlap.

Reply:
@StephenLPeters The `RenderTransforms` are for the indeterminate animation where `Storyboard.TargetProperty="(UIElement.RenderTransform).(CompositeTransform.TranslateX)"`